packages/eslint-config-hop-protocol/index.js

```diff
@@ -0,0 +1,114 @@
+module.exports = {
+  "env": {
+    "node": true,
+    "browser": true,
+    "es2021": true,
+    "jest": true
+  },
+  "extends": [
+    "eslint:recommended",
+    "plugin:json/recommended",
+    // "plugin:@typescript-eslint/recommended-type-checked", // this is causing issues with some packages (core, stats-worker). Need to debug and re-enable
+    "standard"
+  ],
+  "parser": "@typescript-eslint/parser",
+  "parserOptions": {
+    "ecmaFeatures": {
+      "jsx": true
+    },
+    "ecmaVersion": 2021,
+    "sourceType": "module",
+    "project": "./tsconfig.json"
+  },
+  "plugins": [
+    "@typescript-eslint",
+    "node",
+    "unused-imports",
+    "sort-imports-es6-autofix"
+  ],
+  "rules": {
+    // Explicit offs
+    "@typescript-eslint/consistent-type-definitions": 0,
+    "@typescript-eslint/explicit-function-return-type": 0,
+    "@typescript-eslint/method-signature-style": 0,
+    "@typescript-eslint/no-dynamic-delete": 0,
```

Comment:
These are all off by default. `no-dynamic-delete` is on with `strict`. but we don't enable that at this time.

```suggestion
```

Reply:
Resolved wtih https://github.com/hop-protocol/hop/pull/545/commits/19782a3ed6c46210c7cd1fea8dc39b26806c93d9